cabot/cabotapp/views.py

```diff
@@ -836,6 +836,7 @@ class ScheduleUpdateView(LoginRequiredMixin, UpdateView):
     context_object_name = 'schedules'
 
     def get_success_url(self):
+        reset_shifts.delay(self.object.id)
```

Comment:
ok. figured it out by reading through the code.

Correct me if I am wrong. Your need is to fire off the task when the `Schedule` model is saved. There are multiple ways to do this.
1. Override the `save` method. Acceptable practice in django, but not clean. Causes business logic to creep into models.
1. Use django signals - https://docs.djangoproject.com/en/1.10/topics/signals/ - Django signals allow you to hook into model actions. Allows for good clean separation!